lib/ddtrace.rb

```diff
@@ -31,24 +31,9 @@ def self.tracer
     require 'ddtrace/contrib/rails/framework'
 
     module Datadog
-      # Run the auto instrumentation directly after the initialization of the application and
-      # after the application initializers in config/initializers are run
+      # Run the auto instrumentation directly after initializers in
+      # `config/initializers` are executed
       class Railtie < Rails::Railtie
-        config.before_configuration do
-          begin
-            # We include 'redis-rails' here if it's available, doing it later
-            # (typically in initialize callback) does not work, it does not
-            # get loaded in the right context.
-            require 'redis-rails'
```

Comment:
we don't need to import that here, because it will be available after cache initialization (if present)